client/database.go

```diff
@@ -1,96 +0,0 @@
-package client
-
-import (
-	"database/sql"
-	"fmt"
-
-	_ "github.com/go-sql-driver/mysql"
-)
-
-const numTables int = 5
-
-func createTable(numTable int, db *sql.DB) error {
-	stmt := fmt.Sprintf(`CREATE TABLE IF NOT EXISTS User%02d (
```

Comment:
The table names are `User01`, `User02`, ... which makes it seem like we only have 5 users. I think we want them to be `Unsub01`, `Unsub02`, ...